src/rust/runtime/types/ops.rs

```diff
@@ -51,8 +47,57 @@ pub union demi_qr_value_t {
 #[repr(C)]
 pub struct demi_qresult_t {
     pub qr_opcode: demi_opcode_t,
-    pub qr_qd: c_int,
+    pub qr_qd: u32,
     pub qr_qt: demi_qtoken_t,
-    pub qr_ret: c_int,
+    pub qr_ret: i64,
     pub qr_value: demi_qr_value_t,
 }
+
+#[cfg(test)]
+mod test {
+
+    use super::*;
+    use ::std::mem;
+
+    /// Tests if `demi_accept_result_t` has the expected size.
+    #[test]
+    fn test_size_demi_accept_result_t() {
```

Comment:
Again for consistency, we should have these tests return a Result<()> and use demikernel::ensure_eq

Reply:
Well noted.